DevProxy.Abstractions/Utils/MSGraphDbUtils.cs

```diff
@@ -48,61 +49,64 @@ public static async Task<int> GenerateMSGraphDbAsync(ILogger logger, bool skipIf
                 return 1;
             }
 
-            var dbConnection = MSGraphDbConnection;
-            await CreateDbAsync(dbConnection, logger, cancellationToken);
-            await FillDataAsync(dbConnection, logger, cancellationToken);
+            await CreateDbAsync(cancellationToken);
+            await FillDataAsync(cancellationToken);
 
-            logger.LogInformation("Microsoft Graph database successfully updated");
+            _logger.LogInformation("Microsoft Graph database successfully updated");
 
             return 0;
         }
         catch (Exception ex)
         {
-            logger.LogError(ex, "Error generating Microsoft Graph database");
+            _logger.LogError(ex, "Error generating Microsoft Graph database");
             return 1;
         }
 
     }
 
     private static string GetGraphOpenApiYamlFileName(string version) => $"graph-{version.Replace(".", "_", StringComparison.OrdinalIgnoreCase)}-openapi.yaml";
 
-    private static async Task CreateDbAsync(SqliteConnection dbConnection, ILogger logger, CancellationToken cancellationToken)
+    private async Task CreateDbAsync(CancellationToken cancellationToken)
     {
-        logger.LogInformation("Creating database...");
+        _logger.LogInformation("Creating database...");
+
+        var dbConnection = MSGraphDbConnection;
 
-        logger.LogDebug("Dropping endpoints table...");
+        _logger.LogDebug("Dropping endpoints table...");
         var dropTable = dbConnection.CreateCommand();
         dropTable.CommandText = "DROP TABLE IF EXISTS endpoints";
         _ = await dropTable.ExecuteNonQueryAsync(cancellationToken);
 
-        logger.LogDebug("Creating endpoints table...");
+        _logger.LogDebug("Creating endpoints table...");
         var createTable = dbConnection.CreateCommand();
         // when you change the schema, increase the db version number in ProxyUtils
         createTable.CommandText = "CREATE TABLE IF NOT EXISTS endpoints (path TEXT, graphVersion TEXT, hasSelect BOOLEAN)";
         _ = await createTable.ExecuteNonQueryAsync(cancellationToken);
 
-        logger.LogDebug("Creating index on endpoints and version...");
+        _logger.LogDebug("Creating index on endpoints and version...");
         // Add an index on the path and graphVersion columns
         var createIndex = dbConnection.CreateCommand();
         createIndex.CommandText = "CREATE INDEX IF NOT EXISTS idx_endpoints_path_version ON endpoints (path, graphVersion)";
         _ = await createIndex.ExecuteNonQueryAsync(cancellationToken);
     }
 
-    private static async Task FillDataAsync(SqliteConnection dbConnection, ILogger logger, CancellationToken cancellationToken)
+    private async Task FillDataAsync(CancellationToken cancellationToken)
     {
-        logger.LogInformation("Filling database...");
+        _logger.LogInformation("Filling database...");
+
+        var dbConnection = MSGraphDbConnection;

```

Comment:
I don't think we need this assignment anymore since we already have it in a class member